app/controllers/contact_messages_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ContactMessagesController < ApplicationController
+  def new
+    @contact_message = ContactMessage.new
+  end
+
+  def create
+    @contact_message = ContactMessage.new(contact_message_params)
+    if @contact_message.save
+      NewContactMessageMailer.contact_message_notification(
+        @contact_message).deliver_now
+      flash[:success] =
+        "It's good to hear from you. We will respond to your message shortly."
+      redirect_to root_path
+    else
+      flash.now[:error] =
+        "Please review your message, some information is missing."
+      render :new
+    end
+  end
+
+  private
+
+  def contact_message_params
+    params.require(:contact_message).permit(
+      :name,
+      :email,
+      :cell,
+      :message
```

Comment:
@kenyonj @halogenandtoast Whilst I like the organized nature of alphabetizing, I have them mirroring the layout of the form. There was one point where I changed the form but didn't update the params in the controller. I was easily able to see which ones I had missed. 
I would like my herbs and spices alphabetized though!